sdk/identity/identity/src/credentials/authorizationCodeCredential.ts

```diff
@@ -105,6 +107,7 @@ export class AuthorizationCodeCredential implements TokenCredential {
 
     this.clientId = clientId;
     this.tenantId = tenantId;
+    this.allowMultiTenantAuthentication = options?.allowMultiTenantAuthentication;
```

Comment:
This should be after your `typeof redirectUriOrOptions` check on line 112. Right now you'll miss this field if redirect options weren't provided.

Reply:
That is correct! thank you!

Reply:
Moved here: https://github.com/Azure/azure-sdk-for-js/pull/15837/commits/723f82535a8e5f1bd9ec4f33d0efa3455534ce4f